packages/nimble-components/src/wafer-map/template.ts

```diff
@@ -1,4 +1,36 @@
-import { html } from '@microsoft/fast-element';
+import { html, repeat } from '@microsoft/fast-element';
 import type { WaferMap } from '.';
+import type { WaferMapDie } from './types';
 
-export const template = html<WaferMap>``;
+const notReadyTemplate = html<WaferMap>` <div>Loading....</div> `;
+
+const waferTemplate = html<WaferMap>`
```

Comment:
prettier can mess up the `html` template formatting badly, you can disable it like we do for other templates and make the whitespace reasonable, ie https://github.com/ni/nimble/blob/a57a2f77d04f03fa8246fcf10537fe8db61715b4/packages/nimble-components/src/toolbar/tests/toolbar-matrix.stories.ts#L20

Reply:
Applied

Reply:
You need to disable it for the statement using prettier-ignore like the example
https://github.com/ni/nimble/blob/a57a2f77d04f03fa8246fcf10537fe8db61715b4/packages/nimble-components/src/toolbar/tests/toolbar-matrix.stories.ts#L19

Reply:
oh, you meant that - done :)